### DISCUSSION TITLE
ESPResSo++ weekly meeting (4.00 pm, 24. Nov)

### DISCUSSION THREAD

**Discussion by xzhh** [espressopp/espressopp#471] [Team Posts]
Join Zoom Meeting
https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09

Meeting-ID: 926 0512 4006
Kenncode: 120525

**Comment by mrampp** [comment]
are we now back to 4pm?

I thought that two weeks ago we had shifted to 4:30 pm until further notice?
anyway, both 4pm and 4:30pm would be fine with me

best regards,
Markus

On Thu, Nov 24 2022, Zhen-Hao Xu wrote:
> Join Zoom Meeting
> https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09
> 
> Meeting-ID: 926 0512 4006
> Kenncode: 120525
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/orgs/espressopp/teams/discussion/discussions/17
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
best regards / schoene Gruesse,
Markus Rampp

--  
---------------------------------------------------------
Dr. Markus Rampp
Max Planck Computing and Data Facility (MPCDF)
Giessenbachstr. 2, D-85748 Garching bei Muenchen, Germany
Phone: +49-89-3299-1324           FAX: +49-89-3299-1301
E-Mail: ***@***.*** WWW: www.mpcdf.mpg.de
---------------------------------------------------------

**Comment by xzhh** [comment]
> are we now back to 4pm?
> 
> I thought that two weeks ago we had shifted to 4:30 pm until further notice?
> anyway, both 4pm and 4:30pm would be fine with me
> 
> best regards,
> Markus
> …

Yes, we discussed and 4.00 pm is now good to everyone
